.travis.yml

```diff
@@ -41,7 +41,7 @@ matrix:
         - docker-compose logs
         # Build and publish compact image with compiled binary
         - docker build --tag stablex-binary-private --build-arg SOLVER_BASE=163030813197.dkr.ecr.eu-central-1.amazonaws.com/dex-solver:v0.6.4 --build-arg RUST_BASE=rust-binary -f docker/rust/Dockerfile .
-        # StableX e2e Tests (Ganache) - naive solver
+        # StableX e2e Tests (Ganache) - private solver
```

Comment:
Should this be "open solver" considering there is nothing private about it.

Reply:
This isn't the open solver, it's the "standard solver" which requires the private docker image.